pkg/resmgr/control/cpu/cpu.go

```diff
@@ -128,13 +127,12 @@ func (ctl *cpuctl) enforceCpufreqGovernor(class string, cpusIds ...int) error {
 	if _, ok := ctl.classes[class]; !ok {
 		return fmt.Errorf("non-existent cpu class %q", class)
 	}
-	governor := ctl.classes[class].FreqGovernor
-	for cpu := range cpusIds {
-		log.Info(strconv.Itoa(cpu), governor)
-	}
-	log.Debug("enforcing cpu frequency governor %q on %v", governor, cpusIds)
-	if err := utils.SetScalingGovernorForCPUs(cpusIds, governor); err != nil {
-		return fmt.Errorf("Cannot set cpufreq governor %d: %w", governor, err)
+	if ctl.classes[class].FreqGovernor != "" {
+		governor := ctl.classes[class].FreqGovernor
+		log.Debug("enforcing cpu frequency governor %q on %v", governor, cpusIds)
```

Comment:
It would be helpful for debugging to print out the class name, too, like in `enforceCpufreq`.